### ISSUE TITLE
Create a bugfix release 3.0.1

### ISSUE THREAD

**Issue by SirkoS** [ESIPFed/sweet#69]
The current release 3.0 of SWEET still uses the "old" `reprSciUnits.ttl` that was missing several units due to errors in the conversion from rdf/xml to ttl.

In my opinion, until 3.1 is available there should be a bugfix release including the changes made in 843984f620b831614d662a03fd9d2d10dcf727fe 
So people have access to a complete, converted version of SWEET.

**Comment by SirkoS** [comment]
I pulled the files from PR #68 and ran them through our scripts.
Seems in terms of unit related individuals that version is identical to SWEET 2.3

For the `sameAs` relation one entry is missing. Currently our scripts don't show me, which one. I'll dig into that as soon as I find some time.

**Comment by SirkoS** [comment]
I was able to track the missing relation down, but I think, this is an error in SWEET 2.3.

`realmRegion.owl` in SWEET 2.3 includes the following definition:
```xml
  <sreg:LatitudeBand rdf:about="#LowLatitude">
    <owl:sameAs rdf:resource="#Tropical"/>
    <owl:sameAs rdf:resource="#Tropics"/>
    <owl:sameAs rdf:resource="#LowLatitude"/>
     <srela:hasLatitudeBand> 
       <sreg:LatitudeBand>
         <mrela:hasLowerBound rdf:datatype="&xsd;integer">-25</mrela:hasLowerBound>
         <mrela:hasUpperBound rdf:datatype="&xsd;integer">25</mrela:hasUpperBound>
       </sreg:LatitudeBand>
     </srela:hasLatitudeBand>
  </sreg:LatitudeBand>
  <sreg:LatitudeBand rdf:about="#Tropics"/>
  <sreg:LatitudeBand rdf:about="#Tropical"/>
  <sreg:LatitudeBand rdf:about="#LowLatitude"/>
```

So it basically states that 
```Turtle
#LowLatitude owl:sameAs #LowLatitude .
```
This is obviously redundant and removed in your PR.

So from my side PR #68 seems fine and could be the base of a new release.

**Comment by carueda** [comment]
Thanks @SirkoS 

@lewismc @dr-shorthair FYI

**Comment by SirkoS** [comment]
Any update on this?

We would like to reference version 3.0.1 instead of 2.3 for SWEET in our publication, but the deadline is closing in fast and the current official release is missing a lot of units ...

**Comment by carueda** [comment]
@lewismc @dr-shorthair  Given your approval to the previous PR https://github.com/ESIPFed/sweet/pull/66  and the extra verification mentioned here,  I just went ahead and merged PR https://github.com/ESIPFed/sweet/pull/68. 

The cronjob mentioned at https://github.com/ESIPFed/sweet/issues/52#issuecomment-343223781, which runs every 6hr, will take care of propagating this update to `http://sweetontology.net/*` in a little while. 

@SirkoS Note that the new version is 3.1.0 (`owl:versionInfo "3.1.0"`).

**Comment by dr-shorthair** [comment]
That's good.

**Comment by SirkoS** [comment]
Thanks. Can you also add a new release under https://github.com/ESIPFed/sweet/releases with the respective version number ?

**Comment by carueda** [comment]
@SirkoS Done. Thanks for helping us move forward with this.

@dr-shorthair @lewismc :  FYI, as part of this I did the following:
- marked some unassigned closed issues with the 3.1.0 milestone
- I "moved" the remaining open 3.1.0 issues to a new 3.1.1 milestone
- closed the 3.1.0 milestone.
